src/MaterialProperties/MaterialProperties.jl

```diff
@@ -31,7 +31,8 @@ material_properties[:HPGe] = (
     ml = 1.64,
     mt = 0.0819,
     De = 101u"cm^2/s",
-    Dh = 49u"cm^2/s"
+    Dh = 49u"cm^2/s",
+    DT = 78u"K" # Temperature in Einstein's relation for diffusion calculation

```

Comment:
As mentioned above, I would remove `DT` that here and pass the temperature via the detector config file.